db/schema.rb

```diff
@@ -24,7 +24,7 @@
   create_table "game_cards", force: :cascade do |t|
     t.bigint "game_id"
     t.bigint "card_id"
-    t.integer "type"
+    t.integer "category"
```

Comment:
this will display blue, red, assassin, etc ?

Reply:
Yep, on the DB it's an integer because on the model it's an Enum, so Rails does the grunt work of storing the correct integer to represent the word we actually care about.

The integers stored on the DB are the same as the index of the respective Symbol in the array where we declare the Enum:
https://github.com/code-words/codewords-server/blob/44d23adbf062a11b5ce2ec5f66e6d809645e1cbf/app/models/game_card.rb#L7